packages/db/postgres/functions.sql

```diff
@@ -256,11 +256,20 @@ DECLARE
 BEGIN
   uploaded :=
     (
-      SELECT COALESCE(SUM(c.dag_size), 0)
-      FROM upload u
-      JOIN content c ON c.cid = u.content_cid
-      WHERE u.user_id = query_user_id::BIGINT
-      AND u.deleted_at is null
+      SELECT COALESCE((
+        SELECT SUM(dag_size) 
+        FROM (
+          SELECT  c.cid,
+                  c.dag_size
+          FROM upload u
+          JOIN content c ON c.cid = u.content_cid
+          JOIN pin p ON p.content_cid = u.content_cid
+          WHERE u.user_id = query_user_id::BIGINT
+          AND u.deleted_at is null
+          AND p.status = 'Pinned'
+          GROUP BY c.cid,
+                  c.dag_size
```

Comment:
Why do we group by `c.dag_size` here? Will that not mean that if two files have the same dag size that one of them will be ignored?

I tried removing this line, and the tests still pass. Should we remove it?

Reply:
`c.dag_size` is used in the `SELECT`, so it should be there. 
I'm not sure why the test were not failing for you 🤔, you should get `column "c.dag_size" must appear in the GROUP BY clause or be used in an aggregate function`.

> Will that not mean that if two files have the same dag size that one of them will be ignored?
I don't think. The dag_size is a column on `content` and we're already grouping on the primary key `cid` so that adding the `dag_size` shouldn't really changed the returned rows but rather just make it available to the aggreate `SUM`.